src/lev-dashboard/DashboardContent.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styled from 'styled-components';
+import DashboardCounter from "./DashboardCounter";
+
+const CountDiv = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: row;
+`;
+
+const REFRESH = 10 * 1000;
+const FRAMES = 4;
+const DURATION = 1500;
+
+class DashboardContent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todaySearches: props.todaySearches,
+      allTime: props.allTime,
+    };
+    this.animateIDs = [];
+  }
+
+  componentDidMount() {
+    this.timerID = setInterval(() => this.updateCounter(), REFRESH);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.timerID);
+    this.stopAnimation();
+  }
+
+  stopAnimation() {
+    this.animateIDs.forEach(id => clearTimeout(id));
+  }
+
+  animateUpdate(todayOld, todayNew, allOld, allNew) {
+    this.stopAnimation();
+    const todayDiff = todayNew < todayOld ? todayNew : todayNew - todayOld;
+    const allDiff = allNew - allOld;
+    this.setState({
+      todaySearches: todayNew < todayOld ? (todayOld = 0) : todayOld + Math.round(todayDiff / FRAMES),
```

Comment:
I think the ternary could be avoided here by factoring out the `(todayOld = 0)` part to it's own statement.

Reply:
Meh, you still need to do the check, and if you handle it elsewhere, you do a needles bit of arithmetic with a `0`... hardly seems like a win

Reply:
Do you though?

If you set `todayOld` to zero then surely the line can become:
```js
todaySearches: todayOld + Math.round(todayDiff / FRAMES),
```

Reply:
It also seems like `Math.round(todayDiff / FRAMES)` could go into it's own function now that I look at it. That should clean it up a bit.

Reply:
Let's leave this for when we separate this out into different components.